### PR TITLE
Updates header.html for changelog

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

It updates header.html so the changelog menu is updated and accurate

#### Changelog

:cl:   
tweak: updates header.html on the changelog so its accurate
/:cl:

### CODE REVIEW

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Identification is technically a maintainer :)

**Comment by swissloaf** [comment]
> Identification is technically a maintainer :)

done

File: html/templates/header.html [hunk 1]
```diff
@@ -35,16 +35,16 @@
...
-			<font size='2'><b>Coders:</b> Nichlas0010, ThatLing, GrayRachnid, AsV9, Kmc2000, Moja10, Cruix, X-TheDark, Oisin100, TehFlaminTaco, Kn0ss0s, Super3222, ShadowDeath6, Time-Green, Ephemeralis, Alek2ander, FluffySurvivor, Nirnael, Utahclock, Macelarius, ToGWTF, Adam El-Tablawy, Johncena1469, McDonald072, ArgyleSweatshop, Altoids<br></font> 
-			<font size='2'><b>Spriters:</b> Partheo, Kmc2000, Andy, Silvaplex, Dagdamnit, Maya/Larissa<br></font>
-			<font size='2'><b>Mappers:</b> Nichlas0010, GrayRachnid, Matskuman5, Utahclock, Stealthkibbler, MayhemSailor, Yewmi<br></font>
+			<font size='2'><b>Current Headcoders:</b>AsV9<br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
+			<font size='2'><b>Maintainers:</b> Alexkar598, Theos, Fluffe9911, Ktlwjec, McDonald072, Monster860, ThatLing, AshCorr, Partheo, Identification<br></font> 
+			<font size='2'><b>Spriters:</b> Missatessatessy, Kmc2000, Identification, r3d, Meeeeep7, Keekenox, Partheo<br></font>
+			<font size='2'><b>Mappers:</b> Ktlwjec, Monster860, MayhemSailor, DotLyna<br></font>
```
Comment (ThatLing) [inline html/templates/header.html]: The spriters and mappers section is pretty useless as there's no set requirement to be a "mapper" or "spriter".

File: html/templates/header.html [hunk 2]
```diff
@@ -35,16 +35,16 @@
...
-			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation-TG/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Coders:</b> Nichlas0010, ThatLing, GrayRachnid, AsV9, Kmc2000, Moja10, Cruix, X-TheDark, Oisin100, TehFlaminTaco, Kn0ss0s, Super3222, ShadowDeath6, Time-Green, Ephemeralis, Alek2ander, FluffySurvivor, Nirnael, Utahclock, Macelarius, ToGWTF, Adam El-Tablawy, Johncena1469, McDonald072, ArgyleSweatshop, Altoids<br></font> 
-			<font size='2'><b>Spriters:</b> Partheo, Kmc2000, Andy, Silvaplex, Dagdamnit, Maya/Larissa<br></font>
-			<font size='2'><b>Mappers:</b> Nichlas0010, GrayRachnid, Matskuman5, Utahclock, Stealthkibbler, MayhemSailor, Yewmi<br></font>
+			<font size='2'><b>Current Headcoders:</b>AsV9<br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
+			<font size='2'><b>Maintainers:</b> Alexkar598, Theos, Fluffe9911, Ktlwjec, McDonald072, Monster860, ThatLing, AshCorr, Partheo, Identification<br></font>
```
Comment (ThatLing) [inline html/templates/header.html]: We should probably have some forum thread or something to display this.

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Nitpick time, can you alphabetize the lists please, I don't want people to start thinking its in order of importance or something

**Comment by ThatLing** [comment]
As an aside, I compiled this list when we rebased. The order of the list is based on the amount of PRs made (or merged) on the prebase repo

File: html/templates/header.html (hunk 1 above)
Comment (Hopekz) [reply html/templates/header.html]: Didn't DotLyna resign?

Comment (boodaliboo) [reply html/templates/header.html]: wait meeeeep7 sprites ive never seen any

Comment (swissloaf) [reply html/templates/header.html]: > Didn't DotLyna resign?

according to ktl. no

Comment (swissloaf) [reply html/templates/header.html]: > wait meeeeep7 sprites ive never seen any

he does

**Pull request merged**